scripts/apps/search/views/item-preview.html

```diff
@@ -19,9 +19,6 @@
             <li class="nav-tabs__tab" ng-show="isMediaUsed" ng-class="{'nav-tabs__tab--active': vm.current_tab === 'used'}">
                 <button class="nav-tabs__link" ng-click="vm.current_tab = 'used'" translate>Used</button>
             </li>
-            <li class="nav-tabs__tab" ng-show="isAssigned" ng-class="{'nav-tabs__tab--active': vm.current_tab === 'assignment'}">
```

Comment:
Hi @tomaskikutis , @fritzSF why are we removing the assignment tab from the `preview`? It would be good to have assignment preview in the preview pane as well as metadata pane (for open articles).

Reply:
My bad, I shouldn't have removed it from the preview. I'll update the PR.